Fixed Typo in the Export powershell link

glensmall opened the pull request:
The section related to [New-AzureRmSqlDatabaseExport] showed [New-AzureRmSqlDatabaseImport] as the link text but the link directed to the Export CmdLet.  I updated the link text to reflect the correct PowerShell command.

azurecla commented:
Hi __@kmouss__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (kmouss). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

Commit:
Fixed Typo in the Export powershell link

The section related to [New-AzureRmSqlDatabaseExport] showed [New-AzureRmSqlDatabaseImport] as the link text but the link directed to the Export CmdLet.  I updated the link text to reflect the correct PowerShell command.
